docs/root/operations/admin.rst

```diff
@@ -184,15 +184,24 @@ The fields are:
   Histograms will output the computed quantiles i.e P0,P25,P50,P75,P90,P99,P99.9 and P100.
   The output for each quantile will be in the form of (interval,cumulative) where interval value
   represents the summary since last flush interval and cumulative value represents the
-  summary since the start of envoy instance.
+  summary since the start of envoy instance. "No recorded values" in the histogram output indicates
+  that it has not been updated with a value.
   See :ref:`here <operations_stats>` for more information.
 
+  .. http:get:: /stats?usedonly
+
+  Outputs statistics that have been updated with values on demand.
```

Comment:
nit: "on demand" wording is a bit strange, though not sure of the best wording honestly. "Outputs statistics that Envoy has updated (counters incremented at least once, gauges changed at least once, and histograms added to at least once)." Same below.